README.md

```diff
@@ -86,42 +106,32 @@ This step will create an AAD Application and Service Principal in the specified
 
     - Copy the block of JSON from the terminal for the next step.
 
-1. Create and populate a GitHub environment
+3. Create GitHub PATs (access tokens)
```

Comment:
Completely agree, when we create the fork following this PR let's create that org and the service user - there's a story on the backlog to see if we can make this a GitHub App instead of a user PAT as well